examples/noise-js/index.js

```diff
@@ -0,0 +1,357 @@
+import { createLightNode } from "js-waku/lib/create_waku";
+import { utils } from "js-waku";
+import { waitForRemotePeer } from "js-waku/lib/wait_for_remote_peer";
+import {
+  Fleet,
+  getPredefinedBootstrapNodes,
+} from "js-waku/lib/predefined_bootstrap_nodes";
+import { PeerDiscoveryStaticPeers } from "js-waku/lib/peer_discovery_static_list";
+import { Protocols } from "js-waku";
+import * as noise from "@waku/noise";
+import protobuf from "protobufjs";
+import QRCode from "qrcode";
+
+// Protobuf
+const ProtoChatMessage = new protobuf.Type("ChatMessage")
+  .add(new protobuf.Field("timestamp", 1, "uint64"))
+  .add(new protobuf.Field("nick", 2, "string"))
+  .add(new protobuf.Field("text", 3, "bytes"));
+
+main();
+
+async function main() {
+  const ui = initUI();
+  ui.waku.connecting();
+
+  // Starting the node
+  const node = await createLightNode({
+    libp2p: {
+      peerDiscovery: [
+        new PeerDiscoveryStaticPeers(getPredefinedBootstrapNodes(Fleet.Test)),
```

Comment:
I think we might be able to use the prod fleet (`defaultBootstrap: true`) now that nwaku 0.14 has been deployed

Reply:
Indeed we can :)